cocotb/share/makefiles/Makefile.pylib.Msys

```diff
@@ -47,3 +47,20 @@ ifeq ($(PYTHON_LIBDIR),None)
 endif
 
 PYTHON_INCLUDEDIR := $(shell $(PYTHON_BIN) -c 'import distutils.sysconfig; print( distutils.sysconfig.get_python_inc() )')
+
+# For tcl scripts purpose source files paths in form of C:/User/..
+ifneq ($(VERILOG_SOURCES),)
+    VERILOG_SOURCES := $(shell cygpath -m $(VERILOG_SOURCES))
+endif
+ifneq ($(VHDL_SOURCES),)
+    VHDL_SOURCES := $(shell cygpath -m $(VHDL_SOURCES))
+endif
+
+# On Windows there is no support for @rpath so we need to set PATH to find cocotb libraries
+export PATH := $(LIB_DIR):$(PATH)
```

Comment:
PATH is used for libraries on Windows? If that's the case it needs a longer comment please.